web/packages/teleterm/src/mainProcess/mainProcess.ts

```diff
@@ -361,6 +362,25 @@ export default class MainProcess {
       }
     );
 
+    ipcMain.handle(
+      'main-process-open-agent-logs-directory',
+      async (
+        _,
+        args: {
+          rootClusterUri: RootClusterUri;
+        }
+      ) => {
+        const { logsDirectory } = generateAgentConfigPaths(
+          this.settings,
+          args.rootClusterUri
+        );
+        const error = await shell.openPath(logsDirectory);
```

Comment:
This is going to fail if the agent was configured but wasn't started yet:

> Failed to open agent logs directory
>
> Error invoking remote method 'main-process-open-agent-logs-directory': Error: Failed to open path

But I think it's not important to handle this, we'd need to check for the existence of the logs dir first and then return some kind of a bool to indicate the result of this operation (as the error crosses the context bridge) and then display a different notification depending on the result which is… a bit of work.

OTOH, if something does go wrong the user is likely to wander there and attempt to click the button to figure out what's going on and this will simply be another thing which doesn't work.

Maybe we could at least add something like this to the end of the notification: `\n\nNote: the logs directory is created only after the agent process successfully spawns.`

Reply:
Good idea, added that to the notification.